client/src/components/RoundButton.vue

```diff
@@ -1,32 +1,51 @@
 <template>
-    <button type="button" class="roundButton" @click="onClick" :class="{ 'button-active': isClicked }">
+    <button
+        type="button"
+        class="roundButton"
+        @click="onClick"
+        :class="{ 'button-active': isClicked || type === 'light' }"
```

Comment:
This might be confusing if the type is light and it's not activated. Probably best to break this into two classes 'style-light' and 'button-active'. None of this actually matters of course, but if we're going to be nit-picky might as well go all the way.

Reply:
Maybe the whole type property was not really implemented obviously. I added some comments and a new css class for better readability. Basically the idea was to add the option of a light button only when the button is not able to toggle.